exporter/elasticsearchexporter/README.md

```diff
@@ -188,9 +193,7 @@ and `data_stream.namespace`. Instead of serializing these values under the `*att
 they are put at the root of the document, to conform with the conventions of the data stream naming
 scheme that maps these as `constant_keyword` fields.
 
-`data_stream.dataset` will always be appended with `.otel`. It is recommended to use with
-`*_dynamic_index::enabled: true` (e.g. `logs_dynamic_index::enabled`) to route documents to data stream
-`${data_stream.type}-${data_stream.dataset}-${data_stream.namespace}`.
+`data_stream.dataset` will always be appended with `.otel` if [dynamic data stream routing mode](#elasticsearch-document-routing) is active.
```

Comment:
expecting a conflict with https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/38500 . Can be resolved once either PR is merged.